hummingbot/client/command/gateway_command.py

```diff
@@ -496,3 +492,60 @@ async def _gateway_connect(
 
                 # Reload completer here to include newly added gateway connectors
                 self.app.input_field.completer = load_completer(self)
+
+    async def _gateway_connector_tokens(
+            self,           # type: HummingbotApplication
+            connector_chain_network: str = None
+    ):
+        """
+        Allow the user to input tokens whose balances they want to monitor are.
+        This are not tied to a strategy, rather to the connector-chain-network
+        tuple. This has no influence on what tokens the user can use with a
+        connector-chain-network and a particular strategy. This is only for
+        report balances.
+        """
+        with begin_placeholder_mode(self):
+            gateway_connections_conf: List[Dict[str, str]] = GatewayConnectionSetting.load()
+            if connector_chain_network is None:
+                if len(gateway_connections_conf) < 1:
+                    self.notify("No existing connection.\n")
+                else:
+                    connector_df: pd.DataFrame = build_connector_tokens_display(gateway_connections_conf)
+                    self.notify(connector_df.to_string(index=False))
+
+            else:
+                vals: List[str] = connector_chain_network.split('_')  # uniswap_ethereum_kovan
+                if len(vals) == 3:
+                    connector_name: str = vals[0]
+                    chain: str = vals[1]
+                    network: str = vals[2]
+
+                    selected_connection: Optional[Dict[str, str]] = None
+
+                    for connection in gateway_connections_conf:
+                        if connection['connector'] == connector_name \
+                           and connection['chain'] == chain \
+                           and connection['network'] == network:
+                            selected_connection = connection
+                            break
+
+                    if selected_connection is None:
+                        self.notify(f"'{connector_chain_network}' is not available. You can add and review available "
+                                    f"gateway connectors with the command 'gateway connect'.")
+                    else:
+                        current_tokens: str = selected_connection.get('tokens', '')
+                        if current_tokens.strip() != '':
+                            self.notify(f"The 'balance' command for '{connector_chain_network}' currently reports token balances for {current_tokens}.")
+
+                        self.placeholder_mode = True
+                        new_tokens: str = await self.app.prompt(
+                            prompt=f"Enter {connector_chain_network} tokens whose balances you want to monitor in hummingbot for wallet {selected_connection['wallet_address']} (eg. DAI,USDC,WETH) >>> "
+                        )
+                        self.app.clear_input()
```

Comment:
Lets not do an extra prompt so it can be uniform with other command parser and sub-parsers.
Let's allow users to pass token list with the same cmd line as the `gateway connector-tokens`.